ena-submission/src/ena_deposition/create_assembly.py

```diff
@@ -687,49 +703,59 @@ def assembly_table_handle_errors(
     db_config: SimpleConnectionPool,
     config: Config,
     slack_config: SlackConfig,
-    time_threshold: int = 15,
-    time_threshold_waiting: int = 48,
-    slack_time_threshold: int = 12,
+    last_retry_time: datetime | None,
+    submitting_time_threshold_min: int = 15,
+    retry_threshold_hours: int = 4,
+    waiting_threshold_hours: int = 48,
+    slack_retry_threshold_hours: int = 12,
 ):
     """
-    - time_threshold: (minutes)
-    - time_threshold_waiting: (hours)
-    - slack_time_threshold: (hours)
-    1. Find all entries in assembly_table in state HAS_ERRORS or SUBMITTING over time_threshold
-    2. If time since last slack_notification is over slack_time_threshold send notification
+    1. Find all entries in assembly_table in state HAS_ERRORS or SUBMITTING
+        over submitting_time_threshold_min
+    2. If time since last slack notification is over slack_retry_threshold_hours send notification
     """
     entries_with_errors = find_errors_in_db(
-        db_config, TableName.ASSEMBLY_TABLE, time_threshold=time_threshold
+        db_config, TableName.ASSEMBLY_TABLE, time_threshold=submitting_time_threshold_min
     )
     if len(entries_with_errors) > 0:
         error_msg = (
             f"{config.backend_url}: ENA Submission pipeline found "
-            f"{len(entries_with_errors)} entries"
-            f" in assembly_table in status HAS_ERRORS or SUBMITTING for over {time_threshold}m"
+            f"{len(entries_with_errors)} entries in assembly_table in status"
```

Comment:
Should we maybe add the first few accessions to the message so we can quickly see if the list has changed (not just the number)